ATAOS/v3/_labels.yaml

```diff
@@ -0,0 +1,3 @@
+# Labels for recommended settings; a dict of label: config_file
+current: hex_m1_hex_202003.yaml
```

Comment:
I think the names of the files are wrong here should be:

```
current : hex_m1_hex_202003_v3.yaml
constant_hex : hex_m1_202003_constant_hex_v3.yaml
```

They are missing a `_v3`